app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx

```diff
@@ -74,24 +74,32 @@ const messages = defineMessages({
   },
 });
 
-const Tooltip = () => {
-  const { formatMessage } = useIntl();
-  return <div>{formatMessage(messages.tooltip)}</div>;
+const Popover = ({ formatMessage }) => (
+  <div className={cx('popover')}>
+    <div className={cx('title')}>{formatMessage(messages.showFieldsHeader)}</div>
```

Comment:
Can we use withPopover title prop for title?

Reply:
I'm not sure that we can pass localizied title to `ShowWithTooltip`